src/features/licenses/components/LicenseForm/index.jsx

```diff
@@ -125,6 +139,16 @@ export const LicenseForm = ({
     setSelectedCatalogs(filteredCatalogs);
   }, [fields.courses, fields.catalogs, eligibleCourses, catalogsList]);
 
+  useEffect(() => {
+    if (!isMultiCatalogSel && selectedCatalogs.length > 0 && created) {
+      setFields({
+        ...fields,
+        licenseName: selectedCatalogs[0].label,
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why is necessary this disable?

Reply:
lint require add fields and setfields as dependency but this state change and generate an error for too many changes